Instagram_App/Instagram_App/Controller/LikeViewController.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  LikeViewController.swift
+//  Instagram_App
+//
+//  Created by Анастасия Козлова on 17.10.2022.
+//
+
+import UIKit
+
+/// Уведомления пользователя
+final class LikeViewController: UIViewController {
+    // MARK: - Private Constant
+    private enum Constant {
+        static let indentifierFollowText = "follow"
+        static let indentifierLikeText = "like"
+        static let indentifierCommentText = "comment"
+        static let indentifierButtonText = "button"
+        static let todayText = "Сегодня"
+        static let weekText = "На этой неделея"
+        static let monthText = "В этом месяце"
+    }
+    private enum Section {
+        case follow
+        case today
+        case week
+        case month
+    }
+    // MARK: - Private Visual Components
+    private let refreshControl: UIRefreshControl = {
+        let ref = UIRefreshControl()
+        ref.tintColor = .white
+        return ref
+    }()
+    private let allSection: [Section] = [ .follow, .today, .week, .month]
```

Comment:
```suggestion

    private let allSections: [Section] = [ .follow, .today, .week, .month]
```
отступ и массив ниже ui